pyiron/lammps/control.py

```diff
@@ -300,18 +296,12 @@ def calc_minimize(
         ionic_force_tolerance *= force_units
 
         if pressure is not None:
-            if None in np.array([pressure]).flatten():
-                if rotation_matrix is not None and not np.isclose(np.linalg.det(rotation_matrix), 1):
-                    raise AssertionError('Pressure cannot contain None if upper triangle in cell is defined')
-            elif rotation_matrix is not None:
-                pressure = self.pressure_to_lammps(pressure, rotation_matrix)
+            pressure = self.pressure_to_lammps(pressure, rotation_matrix)
             str_press = ""
             for press, str_axis in zip(pressure, [" x ", " y ", " z ", " xy ", " xz ", " yz "]):
                 if press is not None:
                     str_press += str_axis + str(press*pressure_units )
```

Comment:
Wait a second, are we converting pressure units twice? `pressure` should already have the right units coming out of `self.pressure_to_lammps`...

I just double-checked the master branch and it sure looks to me like we're double converting, but I also resonate with Tobias's earlier comment about maybe being sleepy 😅 Is anyone else seeing what I'm seeing?

Reply:
Indeed, I guess this would also get missed in the tests, since there we only check the output of `pressure_to_lammps`, and not the actual pressure that's written to the input field when we set a calc minimize!

Reply:
This is serious, but simple. It's fixed in [this PR](https://github.com/pyiron/pyiron/pull/1090), which should probably be merged before the one under discussion.

Reply:
Ok, nevermind, it is not as simple as I hoped. Merge this first and we'll just live with the fact that non-zero pressure minimizations are using the wrong pressure. I don't think anyone is doing those anyway.

Hopefully once this PR is merged, the problems in that one will be easier to resolve.